docs/usage.md

```diff
@@ -138,4 +138,10 @@ Services can be deleted by clicking on the trash can icon from the main view.
 
 Once you have accepted the message shown in the image above, the service will be deleted after a few seconds.
 
-![function deleted](images/usage/usage-22.png)
\ No newline at end of file
+![function deleted](images/usage/usage-22.png)
+
+
+## MinIO Storage Tab
+
+
+MinIO Storage Tab is made to manage buckets without using MinIO UI. It simplifies the process. From MinIO Storage Tab, buckets can be created or removed and folders inside them. Furthermore, files can be uploaded to the buckets and downloaded from them. Each time a service is created or submitted an edit, the buckets that are not created will be formed.
```

Comment:
@gmolto please check if this explanation would be necessary in this section.

Reply:
I'd keep it. It fits within the overall description.